relayer/pkg/chainlink/txm/txm.go

```diff
@@ -293,17 +294,8 @@ func (txm *starktxm) broadcast(ctx context.Context, publicKey *felt.Felt, accoun
 	// finally, transmit the invoke
 	res, err := account.Provider.AddInvokeTransaction(execCtx, &broadcastTxnV3)
 	if err != nil {
-		// TODO: handle initial broadcast errors - what kind of errors occur?
-		var dataErr *starknetrpc.RPCError
-		var dataStr string
-		if !errors.As(err, &dataErr) {
-			return txhash, fmt.Errorf("failed to read EstimateFee error: %T %+v", err, err)
-		}
-		data := dataErr.Data
-		dataStr = fmt.Sprintf("%+v", data)
-		txm.lggr.Errorw("failed to invoke tx", "accountAddress", accountAddress, "error", err, "data", dataStr)
-
-		if strings.Contains(dataStr, RPCNonceErrMsg) {
+		txm.lggr.Errorw("failed to invoke tx", "accountAddress", accountAddress, "error", err)
+		if strings.Contains(err.Error(), RPCNonceErrMsg) {
```

Comment:
this seems like a clean change, but we should verify that err.Error() has the expected error string we're looking for

Reply:
We verified it during manual testing. `dataStr` was empty, while `err.Error()` had the required error message.